src/Symfony/Component/DependencyInjection/Tests/Loader/FileLoaderTest.php

```diff
@@ -113,6 +114,27 @@ public function testRegisterClassesWithExclude()
         $this->assertFalse($container->has(DeeperBaz::class));
     }
 
+    public function testRegisterClassesWithExcludeAsArray()
+    {
+        $container = new ContainerBuilder();
+        $container->setParameter('sub_dir', 'Sub');
+        $loader = new TestFileLoader($container, new FileLocator(self::$fixturesPath.'/Fixtures'));
+
+        $loader->registerClasses(
+            new Definition(),
+            'Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\\',
+            'Prototype/*', array(
```

Comment:
`FileLoader::registerClasses` docblock must be updated, right?

https://github.com/symfony/symfony/blob/0082981e4135cf6c04545156db8fcda2c9459f9a/src/Symfony/Component/DependencyInjection/Loader/FileLoader.php#L50

Reply:
You're absolutely right, I completely missed this one. My bad

Reply:
Docblock updated